contacts.js

```diff
@@ -3,25 +3,28 @@ const isFeed = require('ssb-ref').isFeed
 // track contact messages, follow, unfollow, block
 
 module.exports = function (sbot, createLayer) {
-  const layer = createLayer('contacts')
+  const updateLayer = createLayer('contacts')
   let initial = false
 
   const INDEX_VERSION = 10
   const index = sbot._flumeUse('contacts2', Reduce(INDEX_VERSION, (g, data) => {
     if (!g) g = {}
 
-    const from = data.value.author
-    const to = data.value.content.contact
-    const value = (data.value.content.blocking || data.value.content.flagged)
-      ? -1
-      : (data.value.content.following === true) ? 1 : -2
+    const source = data.value.author
+    const dest = data.value.content.contact
+    const edgeValue =
```

Comment:
Using `source` and `dest` similar to the names used in `opts.source` and `opts.dest` in `isFollowing`.

`value` => `edgeValue` because it was sometimes ambiguous what is the meaning of the word "value" (e.g. in leveldb indexes there is "key / value" which is a different thing).